interactive_templates/render.py

```diff
@@ -52,8 +52,9 @@ def render_analysis(schema, output_dir):
     readme = output_dir / "README.md"
     # this allows actions to template their own readme if needed
     if not readme.exists():
+        repo_name = "/".join(schema.repo.split("/")[-2:])
```

Comment:
You just want the last part of the URL here, we've hardcoded the owner to `opensafely` so it's just the repo name, eg `interactive_templates`, that's needed here.

Reply:
fixed, thanks